src/app/shared/auth/auth.service.ts

```diff
@@ -50,10 +52,13 @@ export class AuthService implements OnDestroy {
         else if (appConfig.authType === 'tokens') return of(tokenAuthFromStorage());
         else return this.authHttp.refreshCookie(); // will fail if the browser has no cookie
       }),
-      catchError(_e =>
+      catchError(_e => {
         // (3) otherwise, create a temp session
-        this.authHttp.createTempUser().pipe(retry(2))
-      ),
+        const defaultLanguage = this.localeService.currentLang?.tag;
+        // If no default language, the app is in error state, no need to create a temp user.
+        if (!defaultLanguage) return EMPTY;
```

Comment:
Probably that's better to throw an error so that the observable fails instead of completing without emitting anything. It may make debugging easier.

Reply:
Fixed